assets/js/draw.js

```diff
@@ -0,0 +1,54 @@
+const ctx = document.getElementById('canvas').getContext('2d')
```

Comment:
You should use `;` at the end of line.

Reply:
Here is the link - https://tidbitsjs.github.io/Awesome-JavaScript-Projects/

Okay, I will take care of this. ( Semicolons ruins life :) ... Just kidding, I'm not good with ; )